### PR TITLE
Add basic e2e testing

### PR DESCRIPTION
**Pull request by b4l**

### CODE REVIEW

File: cypress/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```
Comment (gberaudo) [inline cypress/tsconfig.json:3]: I guess you should not target es5. Maybe es2017 would be good.

File: cypress/integration/sample_spec.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+describe('The Home Page', () => {
+  it('sucessfully loads', () => {
+    cy.visit('/');
```
Comment (gberaudo) [inline cypress/integration/sample_spec.ts:3]: Could you had one basic check like there is some DOM element we know should be there or something more cool like checking Cesium globe loaded.

Comment (b4l) [reply cypress/integration/sample_spec.ts:3]: Added declining the tracking consent.

Unfortunately i think what takes long is not DOM but content, also the page has infinite reloads in the cypress test environment and requires a hard coded wait to execute the test more stable.

File: cypress/tsconfig.json (hunk 1 above)
Comment (b4l) [reply cypress/tsconfig.json]: Done!

File: cypress/integration/sample_spec.ts (hunk 2 above)
Comment (gberaudo) [reply cypress/integration/sample_spec.ts:3]: Hmm, I see. Definitely not something to enable in CI then.
Maybe Cypress is not completly ready for us yet.

**Review by gberaudo — APPROVED** [review]

**Pull request merged**